docs/src/standard_results/depth_surfaces.md

```diff
@@ -0,0 +1,44 @@
+# Modelled depth prediction surfaces
+
+This exports the modelled depth prediction surfaces from within RMS.
+These surfaces typically represent the final surface set generated during a structural
+modeling workflow (after well conditioning), and serve as the framework for constructing the grid.
+
```

Comment:
Could perhaps add:
Only one set of depth surface predictions are exported per model workflow, representing the structural prediction of the model.